validator/src/main/java/com/amazon/aoc/validators/CWLogValidator.java

```diff
@@ -36,7 +36,7 @@
 
 @Log4j2
 public class CWLogValidator implements IValidator {
-  private static int DEFAULT_MAX_RETRY_COUNT = 15;
+  private static int DEFAULT_MAX_RETRY_COUNT = 20;
```

Comment:
Can we drop the metric validation retry count down from 80 to something more reasonable? Mostly unrelated to this change but worth keeping our limits in check

Reply:
Also reduced it to 30. Should be 6 minutes ish now